client/app/core/core.module.js

```diff
@@ -1,45 +1,75 @@
 /* global _:false, ActionCable:false, $:false, sprintf: false, moment: false */
+/* eslint-disable sort-imports */
```

Comment:
The import order shouldn't matter here, I added the disable clause for now because I already went through and separated out the services per feature module to make the follow-up PR easier.